modules/serverless-logic-upgrade-deleting-workflows-with-dev-profile.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-upgrading-operator-from-1.34-to-1.35
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-logic-upgrade-deleting-workflows-with-dev-profile_{context}"]
+= Deleting workflows with `dev` profile
+
+Before upgrading the Operator, you must delete workflows running with the `dev` profile and redeploy them after the upgrade is completed.
+
+.Procedure
+
+. Ensure you have a backup of all necessary Kubernetes resources, including `SonataFlow` custom resource definitions (CRDs), `ConfigMaps`, or any other related custom configurations.
+
+. Delete the workflow by executing the following command: 
++
+[source,terminal]
+----
+$ oc delete -f <my-workflow.yaml> -n <target_namespace>
+----
+
+
```

Comment:
I'll just mark this once, but I have been told to remove empty lines as it can sometimes cause issues with the build. It seems to be OK with these files at least by looking at the previews, but might be a good practice to follow.